code-challenges.test.js

```diff
@@ -14,31 +14,108 @@
+
+// // Test Suites: 1 passed, 1 total
+// // Tests:       1 passed, 1 total
+
+// // --------------------2) Create a function that takes in an array of words and a single letter and returns an array of all the words containing that particular letter.
+
+// // a) Create a test with expects statement using the variable provided.
+
+// describe("containingLetter", () => {
+//     it("returns an array of all the words containing that particular letter", () => {
+
+//      const fruitArray = ["Mango", "Cherry", "Apricot", "Blueberry", "Peach", "Kiwi"]
+//      const letterA = "a"
+//     const expectedOutput1 = ["Mango", "Apricot", "Peach"]
+//     const letterE = "e"
+//     const expectedOutput2 = ["Cherry", "Blueberry", "Peach"]
+
+//       expect(containingLetter(fruitArray, letterA)).toEqual(expectedOutput1)
+//       expect(containingLetter(fruitArray, letterE)).toEqual(expectedOutput2)
+
+//     })
+// })
+//     // created a function takes an array and str as parameters 
+//   const containingLetter = (arr, str) => {
+//     // use .map()method to lowercase all words in the array
+//         let tolowerCaseArray = arr.map((value => value.toLowerCase()))
+//         //use .filter() method to find words in that contain identified string 
+//         let searchstrArray   = tolowerCaseArray.filter((value => value.includes(str)))
+//    //  use .map to uppercase the letter of each value which are lowercased
+//    // and used .substr() method to provide rest letters in lowercased.
+//         let toupperCaseArray = searchstrArray.map((value => value[0].toUpperCase() + value.substr(1)))
+//         return toupperCaseArray
+//     // Stored each iteration into its own variable with stored information.
+//     // Each iteration references previous array.     
+//   }
+ 
 // b) Create the function that makes the test pass.
 
-// --------------------2) Create a function that takes in an array of words and a single letter and returns an array of all the words containing that particular letter.
+// Test Suites: 1 failed, 1 total
+// Tests:       1 failed, 1 total
 
-// a) Create a test with expects statement using the variable provided.
+// 27 Nov.--> Test Suites: 1 passed, 1 total
+//            Tests:       1 passed, 1 total
 
-const fruitArray = ["Mango", "Cherry", "Apricot", "Blueberry", "Peach", "Kiwi"]
+// --------------------3) Create a function that takes in an array of 5 numbers and determines whether or not the array is a "full house". A full house is exactly one pair and one three of a kind.
 
-const letterA = "a"
-// Expected output: ["Mango", "Apricot", "Peach"]
-const letterE = "e"
-// Expected output: ["Cherry", "Blueberry", "Peach"]
+//a) Create a test with expect statements using the variable provided.
+describe("fullcards",() => {
+  it('determines whether or not the array is a "full house"',() => {
 
-// b) Create the function that makes the test pass.
+      
+     expect(fullcards(hand1)).toEqual([5, 5, 5, 3, 3])
```

Comment:
Make sure your test is checking what output you actually want to get, which in this case should be a boolean value.